### ISSUE TITLE
Throbber.js deprecation message ...

### ISSUE THREAD

**Issue by rufuspollock** [Scifabric/crowdcrafting.org#17]
@teleyinex great to see the deprecation message but some thoughts:
- Mis-spelling of throbber.js in the message (it is throbber not trobber?)
- Not sure having this on all pages is useful - if i am a general user i'm just a bit confused. Any way we could just show this to owners on their admin page?
- Overall I'm not sure about notifications like this on the site - wouldn't it be better just to email all app owners (getting their email addresses should not be hard -- we do have emails addresses for everyone right?).

My main point here is that most users of the site are not app owners and putting up a message like this just worries a whole bunch of people it is not relevant too ...

**Comment by teleyinex** [comment]
- Mis-spelling has been fixed! Thanks!
- The announcement system has different type of users: [admin, owners, and general users](http://docs.pybossa.com/en/latest/install.html#global-announcements-for-the-users). This message is only shown to users who have created an application, otherwise they will not see anything (unit tests included!).

I've implemented this because some time ago you mentioned this idea and I liked. In general I think is a good solution, several systems use this like Trello, and as we have a great detail of who is going to see the message is great. Regarding the e-mails: if a user has sign up using the Twitter system, we do not have their e-mail as Twitter does not share it, we ask the user to add it after signing up, but sometimes they do not fill it. Thus, this is more like a system warning that it is shown only for specific people and that will be there for long announcements. We can use it also with the e-mail solution.

Thus, in order to answer your broader question: we support different layers of users and it is in the [documentation](http://docs.pybossa.com/en/latest/install.html#global-announcements-for-the-users).
